models/fetchAllArticles.model.js

```
@@ -0,0 +1,31 @@
+const db = require("../db/connection.js");
+
+function fetchAllArticles(sort_by = "created_at", order = "DESC") {
+  const validCols = ["comment_count", "created_at", "article_id", "title", "votes", "author", "topic"];
+  const validOrder = ["ASC", "DESC"];
+
+  if (!validCols.includes(sort_by)) sort_by = "created_at";
+  if (!validOrder.includes(order)) order = "DESC";
```

Probably want to throw an error message here